tests/test_conversion.py

```diff
@@ -86,18 +86,36 @@ def test_enum_with_class_converted_to_enum_field(self, models):
 
     def test_many_to_many_relationship(self, models):
         student_fields = fields_for_model(models.Student, include_relationships=True)
-        assert type(student_fields["courses"]) is RelatedList
+        courses_field = student_fields["courses"]
+        assert type(courses_field) is RelatedList
+        assert courses_field.required is False
 
         course_fields = fields_for_model(models.Course, include_relationships=True)
-        assert type(course_fields["students"]) is RelatedList
+        students_field = course_fields["students"]
+        assert type(students_field) is RelatedList
+        assert students_field.required is False
 
     def test_many_to_one_relationship(self, models):
         student_fields = fields_for_model(models.Student, include_relationships=True)
-        assert type(student_fields["current_school"]) is Related
+        current_school_field = student_fields["current_school"]
```

Comment:
This is the case by OP in #336 . `Student.current_school_id` has `nullable=False`

https://github.com/marshmallow-code/marshmallow-sqlalchemy/blob/ffe1f195eb55b1353e07a0c245e00c8ce0e72c07/tests/conftest.py#L112-L115